Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM postfwd/postfwd:v1.37
+
+LABEL maintainer="Postfwd GeoIp Spam Plugin Maintainer <ondrej.vaskoo@gmail.com>"
+
+ENV PORT=10040
```

Comment:
unused ENV variables?

Reply:
I wanted to assign it to port expose, but probably forgot it. 

But is it actually worth using the `ENV` for `EXPOSE`? I think I have seen this somewhere, but it seems useless.

Reply:
i don't see usecase for having option to change containers port. Anyway it can be set with `--expose` option. It's one extra layer that doesnt bring any benefit.

Reply:
Done in rebase